client/src/components/CenterpieceDropDown.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+
+const CenterpieceDropDown = (props) => {
+  console.log('this is friends prop: ', props.friends)
+  const clocks = props.friends.map((friend) => {
+      console.log('this is a friend in drop: ', friend)
```

Comment:
Your semicolon line ending usage is inconsistent. You should highly consider using eslint to check all kinds of common syntax issues and to help you conform to a consistent style. See https://eslint.org/docs/user-guide/getting-started